src/base-styles/helperClasses/border/sides.css

```diff
@@ -0,0 +1,54 @@
+/*---
+title: Borders by side
+name: borderDirections
+section: Border Helpers
+---
+
+Adds standard borders to an element for a given side.
+
+class            | description
+---------------- | ---------------------------------
+`border--none`   | removes all borders
+`border--top`    | applies a standard top border
+`border--right`  | applies a standard right border
+`border--bottom` | applies a standard bottom border
+`border--left`   | applies a standard left border
+`border--all`    | applies a standard border to all sides
+*/
+
+:root {
+  --standardBorder: 1px solid rgba(var(--rgb-charcoal), 0.12);
+}
+
+.border--none {
+  border-width: 0;
+}
+
+.border--top {
+  border-top: var(--standardBorder);
+}
+
+.border--right {
+  border-right: var(--standardBorder);
+}
+
+.border--bottom {
+  border-bottom: var(--standardBorder);
+}
+
+.border--left {
+  border-left: var(--standardBorder);
+}
+
+.border--all {
+  border: var(--standardBorder);
+}
+
+.inverted:matches(
+.border--top,
+.border--right,
+.border--bottom,
+.border--left,
+.border--all,) {
+  border-color: var(--font-color-hintInverted);
```

Comment:
Should be be using `font-color` to style `border-color`?

Reply:
In this case, yes - we're taking advantage of the alpha transperency of hint text for inverted color sections